scripts/build.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
+# Update repo
+git checkout master
+git pull -q --ff-only
+git submodule update --init
```

Comment:
I usually add `--recursive` here too. It's not required, but a nice just-in-case.